documentation/optee_design.md

```diff
@@ -451,6 +451,36 @@ OP-TEE to be able to initialize at all. The loader supplies in `r0/x0` the
 address of the first byte following what was not copied and jumps to the load
 address to start OP-TEE.
 
+Extra image files are generated by loaders who support loading separate images.
+Legacy OP-TEE image tee.bin is split into several v2 images; typically header
+image, pager image and pageable image. Loaders load header image first to get
+image list and information of each image; and then load each of them into
+different load address assigned in structure; finally main image binary
+(`pager` in general) gets launched after all images loaded.
```

Comment:
I think maybe we don't need to distinguish the main image and secondary image. It will confuse the reader. So I will delete this sentence.